setup.cfg

```diff
@@ -32,6 +32,9 @@ filterwarnings =
     ignore:Importing from numpy.testing.decorators is deprecated, import from numpy.testing instead:DeprecationWarning
     ignore:np.loads is deprecated, use pickle.loads instead:DeprecationWarning
     ignore:The oldnumeric module will be dropped:DeprecationWarning
+    ignore:Collection picker None could not be converted to float:UserWarning
+    ignore:covariance is not positive-semidefinite:RuntimeWarning
+    ignore:Can only plot ICA components:RuntimeWarning
```

Comment:
the matplotlib warning is missing 
`Matplotlib is building the font cache using fc-list`

I'm not sure if we should ignore it here or require a version, like with `sklearn`. Is there a way to change the setup depending on the versions?

Reply:
maybe some fancy `@pytest.mark.filterwarnings("ignore:api v1")` depending on the version. But I'm not sure if its worth it. On the other hand, ignore them always might not be a good strategy either.